sickbeard/common.py

```diff
@@ -212,7 +212,7 @@ def sceneQuality(name, anime=False):
             return Quality.SDTV
         elif checkName(["web.dl|webrip", "xvid|x264|h.?264"], all) and not checkName(["(720|1080)[pi]"], all):
             return Quality.SDTV
-        elif checkName(["(dvdrip|b[r|d]rip)(.ws)?.(xvid|divx|x264)"], any) and not checkName(["(720|1080)[pi]"], all):
+        elif checkName(["(dvdrip|b[rd]rip|blueray)(.ws)?.(xvid|divx|x264)"], any) and not checkName(["(720|1080)[pi]"], all):
```

Comment:
Its a character group, you don't need an |.
`[this]` would match a single character, one of `t`,`h`,`i`, OR, `s`